vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+- Intelligent cohort replacement rate harvest (CRR)
+- Age selective harvest 
+- Dry year harvest regulations 
+- Hatchery harvest only (ocean and/or tributary)
+
+For more information on these harvest strategies please see [DSM scenario harvest actions](https://reorienting-to-recovery.github.io/R2Rscenarios/articles/harvest-actions.html). 
+
+### Adult Enroute Migratory Survival 
+
+Adult enroute migratory survival is a function of migratory temperatures and whether or not the bypass is overtopped. This function also contains a calibrated intercept.  In the model, this information is contained in the following function input objects: 
+
+- data: `migratory_temp`
+- data: `bypass_overtopped`
+- coefficent: `.adult_en_route_bypass_overtopped`
+- coefficent: `.adult_en_route_migratory_temp`
+- calibrated intercept: `..surv_adult_enroute_int` 
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+*Source*
+
+- `.enroute migratory_temp` coefficent from Schreck et al. (1994)
+- `.adult_en_route_bypass_overtopped`coefficient for bypass_overtopped variable, source: Expert opinion Ted Sommer, California Department of Water Resources (tributaries above bypasses only)
+- *Calibrated intercept.* 
+
+### Adult Stray 
+
+The adult straying function is a beta regression model that including the following factors that straying rates:  transport distance, return year flows, fish age, release timing, and environmental conditions (PDO). This logic was developed by Sturrock et al. (2019). Within the DSM, the `compute_adult_stray_rates` function, takes in: 
+
+- `run_year` year of run
+- `age` age of fish
+- `released` total number of fish released at hatchery
+- `flow_oct_nov` the median flow for October and November
+- `flow_apr_may` the median flow for April and May
+- `mean_pdo_return` environmental conditions 
+
+And based on the location of fish assigns a transport distance to bay, and then computes a stray rate for each release group. 
+
+*Source*
+[Sturrock et al. (2019)](https://afspubs.onlinelibrary.wiley.com/doi/epdf/10.1002/fsh.10267) Examining eight decades of hatchery release data in California’s Central Valley.
+
+Please see the [stray rate documentation](https://reorienting-to-recovery.s3.us-west-1.amazonaws.com/updated-stray-rates.html) for more information on how Sturrock et al. (2019) stray logic is incorporated into the R2R DSMs. 
+
+### Adult Prespawn Survival
+Adult prespawn survival is a function of migratory temperatures and whether or not the bypass is overtopped. This function also contains a calibrated intercept.  In the model, this information is contained in the following data objects: 
+
+- data: `degree_days`
+- data: `month_return_proportions`
+- coefficent: `.adult_prespawn_deg_day`
+- calibrated intercept: `..surv_adult_prespawn_int` 
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+*Source*
+
+- `.adult_prespawn_deg_day` coefficent from Colvin et al. (2018)
```

Comment:
same comment re: either finding a link to this, or including the paper name?

Reply:
We also don't have this one...